stories/virtru/sidebar/basic.jsx

```diff
@@ -0,0 +1,65 @@
+import 'antd/dist/antd.less';
+import { storiesOf } from '@storybook/react';
+import React, { useState } from 'react';
+import { Sidebar, SidebarCard } from '@';
+import { CloseOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+
+storiesOf('virtru/sidebar', module).add(
+  'sidebar',
+  () => {
+    const [visibleDrawer, setVisibleDrawer] = useState(false);
+    const card1 = {
+      index: 1,
+      title: 'Session Information',
+      extra: (
+        <CloseOutlined
+          style={{ fontSize: '23px', color: 'var(--vds-color-slate-darker)', marginLeft: '10px' }}
+          onClick={() => {
+            console.log('click');
+            setVisibleDrawer(false);
+          }}
+        />
+      ),
+      cover: <div>Card body JSX</div>,
```

Comment:
Cover is not for the card body, so I think we also need to rename this prop. AntD use `children` for the card content and we should too because that is common in React. See the first example: https://ant.design/components/card/

Reply:
fixed